### ISSUE TITLE
specific methods about the stage two.

### ISSUE THREAD

**Issue by uzabon** [microsoft/ProDA#8]
After reading your paper and amazed by model performance, I have problem in understanding the specific methods  in stage two. I  know little about representation learning. And the paper doesn't explain this table in detail so I hope someone can give me an explanation.
![image](https://user-images.githubusercontent.com/25972051/116398272-31258e80-a85a-11eb-9faa-ff38c15f8101.png)

In stage two, what does it means by self-supervised initialization? And how to combine the self-supervised initialization with/without self distillation? 

Here are my own understanding for stage two.
These three initialization in stage two represents the methods of initialization of student model and the teacher model  is always the best-performing model in stage one if self-distillation is used. Stage1 init. is  the best-performing model in stage one(that is 53.7 in stage one). Supervised init. is the SimCLRv2 pretrained model(I assume resnet101 with supervised training).  

I don't know if my understanding is right.

**Comment by super233** [comment]
I think the SimCLRv2 pretrained model is not **supervised init.** but **self-supervised init.**

**Comment by panzhang0212** [comment]
Supervised init is backbone pretrained on ImageNet by classification label (Cross Entropy loss)
Self-supervised init is backbone pretrained on ImageNet by SimCLRv2 (Contrastive loss, no label used)
